src/components/organization.js

```diff
@@ -1,10 +1,42 @@
-import React from 'react';
+import React from 'react'
 
-const Organization = () => (
-  <footer>
-    <h3>Patrocina y Organiza</h3>
-    <p>Seccion Voluntarios</p>
-  </footer>
+import OrganizationItem from './organizationItem'
+import data from '../data'
+import '../styles/organization.scss'
+
+const organization = () => (
+  <section>
```

Comment:
Moviendo el componente de abajo ya este `section` sería innecesario, o podrías tener: 
```suggestion
  <section className="sponsors-container">
```

Reply:
Me parece bien, yo hago el cambio con el del comentario anterior.